### ISSUE TITLE
Is there a way to grab the event dates after its creations?

### ISSUE THREAD

**Issue by marciok** [vonovak/react-native-add-calendar-event#60]
Hi, I'm trying to get event dates after its creation, as far as I understood there's no way of doing that, on iOS for instances the dates are not passed on the delegate. Is that correct, or there is a workaround? 

Thanks!!

**Comment by vonovak** [comment]
hi! this is indeed something that makes sense, feel free to open a PR for this. On ios, this should be fairly easy to do here https://github.com/vonovak/react-native-add-calendar-event/blob/2f6ea55e27b0eb08bb763cd4d7ead905b4b8ee2b/ios/AddCalendarEvent.m#L304

**Comment by marciok** [comment]
Exactly, on iOS is pretty simple, what about on android? I looked at the code and honestly I have no idea how to do it, or if that is even possible  :(

Thank you!

**Comment by vonovak** [comment]
you could use https://github.com/wmcmahan/react-native-calendar-events/blob/2d6f2416ba1eff3769f41b1237904c37e69a4ef6/android/src/main/java/com/calendarevents/CalendarEvents.java#L349 given the event id
